### PR TITLE
:sparkles: gdpr management

### PR DESCRIPTION
**Pull request by ntchou**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
         "i18next-browser-languagedetector": "^4.0.2",
         "i18next-xhr-backend": "^3.2.2",
         "libphonenumber-js": "^1.7.49",
+        "lodash": "^4.17.15",
```
Comment (HugoGresse) [inline package.json:36]: lodash is very heavy, we could probably go without it without major issue don't you think?

Comment (ntchou) [reply package.json:36]: Yes it was for simplicity. I'll remove it. I wanted to use optional chaining but I don't know if it's already ready. I saw it didn't work on Firefox android

Comment (HugoGresse) [reply package.json:36]: maybe CRA pollyfil it?

File: src/utils/cookies.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const hasCookiebot = () => window.Cookiebot
+
+const cookieBotAcceptListener = () => {
+    if (hasCookiebot() && window.Cookiebot.consent?.statistics) {
+        firebase.analytics()
+    }
+}
+
+const cookieBotDeclineListener = () => {
+    if (hasCookiebot() && window.Cookiebot.consent?.statistics) {
```
Comment (HugoGresse) [inline src/utils/cookies.js:13]: this is crazy /o/ 
what a day we leave in.

File: src/utils/cookies.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import firebase from 'firebase/app'
+import 'firebase/analytics'
+
+const hasCookiebot = () => window.Cookiebot
```
Comment (HugoGresse) [inline src/utils/cookies.js:4]: Having Google anaytics logic within a cookie.js make it weird, I'll have either split that in two files (exporting it from the firebase.js for eg) or have an analytics.js with the cookie logic inside or something like this.

File: src/utils/cookies.js (hunk 2 above)
Comment (ntchou) [reply src/utils/cookies.js:13]: mistake

**Pull request merged**